docs/_static/switcher.json

```diff
@@ -1,51 +1,34 @@
 [
   {
-    "name": "v0.7.1 (stable)",
-    "version": "0.7.1"
+    "name": "dev",
+    "version": "latest"
   },
   {
-    "version": "0.7.0"
+    "name": "0.8.0 (stable)",
+    "version": "stable"
```

Comment:
So the problem here is that, in order to "match" this version, you would need to set the variable to "stable" in ``conf.py``, but I have no idea how to know it during the build on readthedocs whether it is building for "stable" or for "0.8.0" (I don't directly find docs on whether they have some environment variable for this or so)

Reply:
@jorisvandenbossche it's unclear to me whether this is a blocker for this PR. Can you clarify?

Reply:
I don't think it needs to be a blocker. The same issue already exists, and the current changes in this PR are already _some_ improvement

Reply:
I think that it does exist in RTD, see how their theme uses the current version here: https://github.com/readthedocs/sphinx_rtd_theme/blob/d64dadf1ceec4f9ff6c1ca2d3ea4c3d0fdb0e8d2/sphinx_rtd_theme/versions.html#L4-L8

Would you like me to merge this PR as-is, or fix this up and then merge?

Reply:
From the docs: https://docs.readthedocs.io/en/stable/builds.html#build-environment

`READTHEDOCS_VERSION | The RTD name of the version which is being built | latest`

It seems we may use `READTHEDOCS_VERSION` env variable... cc @jorisvandenbossche

Reply:
Ah, yes, it was `READTHEDOCS_VERSION` that I was looking for!